bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml

```
@@ -84,7 +84,79 @@ metadata:
                 "name": ""
               },
               "serviceMonitor": {
-                "enabled": true
+                "enabled": true,
+                "metricRelabelings": [
+                  {
+                    "action": "replace",
+                    "regex": "DCGM_FI_DEV_GPU_UTIL",
+                    "replacement": "accelerator_gpu_utilization",
+                    "sourceLabels": [
+                      "__name__",
```

The comma needs to be removed since it is the last element in the array.
```suggestion
                      "__name__"
```

---

fixed